versions.tf

```diff
@@ -1,6 +1,3 @@
 terraform {
   required_version = ">= 0.13"
-  required_providers {
```

Comment:
Why would you remove it entirely instead of loosing it. If this works with a 4.x great, just change this to `>= 3.36.0, < 5.0.0`

Reply:
this module only renders text. It doesn't actually interact with AWS.